benchmark/buffers/buffer-read.js

```diff
@@ -30,14 +30,14 @@ function main(conf) {
   var len = +conf.millions * 1e6;
   var clazz = conf.buf === 'fast' ? Buffer : require('buffer').SlowBuffer;
   var buff = new clazz(8);
-  var fn = 'read' + conf.type;
+  var fn = `read${conf.type}`;
```

Comment:
At this point I don't think it really matters. I can't see anyone wanting to run the benchmarks for v0.10 or v0.12 anymore, especially since it wouldn't really do any good (it's not as if V8 can/will revert to v0.10/v0.12-era V8 code if there is a performance issue in node v4.x+).